app/Http/Controllers/Web/ActionsController.php

```diff
@@ -73,7 +73,7 @@ public function store(Request $request)
             info('action_created', ['id' => $action->id]);
         }
 
-        return redirect()->route('actions.show', ['id' => $action->id]);
+        return redirect('actions/' . $action->id);
```

Comment:
Does the `update` route below need to be updated as well?

Reply:
Great catch - thanks! 🎣 Fixed the update routes as well.